docs: add stage0-finishers.md (#312)
kottans/frontend: students/stage0-finishers.md

a table row below for yourself following the example.

If you completed mandatory tasks only, no problem, go ahead.
You may change your record if you decide to complete
optionals later.

_Make sure the file contains an empty line at the end._
_When editing file via GitHub add two newlines at the file end._

Fields description:
 * **Date** is the date you have completed mandatory tasks
   (the date of the latest commit to the most recent practical task PR merged,
   if e.g. your latest practical task was **Friends App**,
   the latest commit to the task-related PR is dated October 12, 2020,
   and the PR was approved on October 15, 2020,
   then the **Date** is `2020-10-12`)
 * **Course repo** is your nick name with a link to your
   course repo (`kottans-frontend`)
 * **R** means **Regulars** and is a number of regular tasks completed;
   normally says `6`; if `5` or lesser then you can
   be inquired for the reasons
 * **P** means **Practicals** and is a number of practical tasks completed;
   normally says `7`; if `6` or lesser then give a reason in **Comment**
 * **O** means **Optionals** and is a number of optional tasks
   completed;

>    `0` is OK; `2` qualifies you for a special badge

Reviewer: ```suggestion
   `0` is OK; `2` is good for a student as optional tasks offer more knowledge
```

>    provided all mandatory tasks are completed

Reviewer: ```suggestion
```

>  * **B** means **Badges** and is a number of badges earned during

Reviewer: ```suggestion
```

>    the course Stage 0; `0` is OK; anything above that is a plus

Reviewer: ```suggestion
```

>  * **Comment** is for optional comments and reasons to skip any mandatory

Reviewer: ```suggestion
 * **Student's Comment** is for optional comments and reasons to skip any mandatory
```

   tasks

>  * **Performance** is intended for mentors team use, e.g.

Reviewer: ```suggestion
 * **Mentors' comment** is intended for mentors team use
```

>    to specify performance badges granted

Reviewer: ```suggestion
```

Use the first table row as a template.

## Finishers

> |  |    Date    | Course repo    | R | P | O | B | Comment | Performance |

Reviewer: ```suggestion
|  |    Date    | Course repo    | R | P | O | Student's Comment | Mentors' Comment |
```

> |--| ---------- | -------------- | - | - | - | - | ------- | ----------- |

Reviewer: ```suggestion
|--| ---------- | -------------- | - | - | - | ------- | ----------- |
```

> |1 | 2020-10-27 | [Shramkoweb](https://github.com/Shramkoweb/kottans-frontend) | 6 | 7 | 0 | 0 | --- | |

Reviewer: ```suggestion
|1 | 2020-10-27 | [Shramkoweb](https://github.com/Shramkoweb/kottans-frontend) | 6 | 7 | 0 | --- | |
```
